src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -827,6 +848,17 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
             set mutationCountBy(m: MutationCountBy) {
                 this._mutationCountBy = m;
             },
+            get structuralVariantCountBy() {
+                if (this._structuralVariantCountBy === undefined) {
+                    // default
+                    return StructuralVariantCountBy.VariantType;
+                } else {
+                    return this._structuralVariantCountBy;
+                }
+            },
+            set structuralVariantCountBy(m: StructuralVariantCountBy) {
```

Comment:
Very minor suggestions:
```suggestion
            set structuralVariantCountBy(s: StructuralVariantCountBy) {
```